docs/release-process.md

```diff
@@ -165,23 +167,24 @@ To: dev@pulsar.apache.org
 Subject: [VOTE] Pulsar Node.js Client Release 1.X.0 Candidate 1
 
 Hi everyone,
-Please review and vote on the release candidate #1 for the version 1.X.0, as follows:
-[ ] +1, Approve the release
-[ ] -1, Do not approve the release (please provide specific comments)
 
 This is the first release candidate for Apache Pulsar Node.js client, version 1.X.0.
 
 It fixes the following issues:
 https://github.com/apache/pulsar-client-node/milestone/1?closed=1
 
 Please download the source files and review this release candidate:
-- Review release notes
-- Download the source package (verify shasum and asc) and follow the README.md to build and run the Pulsar Node.js client.
+- Download the source package, verify shasum and asc
+- Follow the README.md to build and run the Pulsar Node.js client.
+
+The release candidate package has been published to the npm registry:
+https://www.npmjs.com/package/pulsar-client/v/1.X.0-rc.1
+You can install it by `npm i pulsar-client@1.X.0-rc.1 --pulsar_binary_host_mirror=https://dist.apache.org/repos/dist/release/pulsar/pulsar-client-node/` and verify the package.
```

Comment:
It seems that the mirror url should be
dist.apache.org/repos/dist/ **dev** /pulsar/pulsar-client-node/

Reply:
You're right. Thanks.